src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -602,5 +602,11 @@ private static string GetFileLookupKey([NotNull] string fileName)
             return Path.GetFullPath(fileName);
 #endif
         }
+
+        /// <inheritdoc />
+        public override string ToString()
+        {
+            return $"{base.ToString()}, XmlFile={_originalFileName}";
```

Comment:
proposal: "config file" or "xml config file path" instead of XmlFile

Reply:
Changed to FilePath